lib/views/name.erb

```diff
@@ -0,0 +1 @@
+<% User.create(params[:name]) %>
```

Comment:
1. This template is not rendered by any action in the controller
2. You should NOT do a db call from the view but rather display any data you pass on to the view from the controller.